CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ## Unreleased
 
+### 💡 Enhancements 💡
+
+- `cmd/mdatagen`:disk_scraper_windows.go Add resource attributes definition to metadata.yaml and move `pdata.Metrics` craetion to the 
```

Comment:
```suggestion
- `cmd/mdatagen`: Add resource attributes definition to metadata.yaml and move `pdata.Metrics` creation to the 
```

Reply:
Good catch. Thanks :)

Reply:
These changes appear not to have actually been applied.

Reply:
Oh I overridden them locally. Let me recover it

Reply:
fixed